_posts/2021-5-12-My-OTP-24-Highlights.md

```diff
@@ -0,0 +1,546 @@
+---
+layout: post
+title: Erlang/OTP 24 Highlights
+tags: erlang otp 24 release
+author: Lukas Larsson
+---
+
+Finally Erlang/OTP 24 is here! A release that for me has been [about 10 years]
+in the making. [As] is [tradition] by [now], this blog post will go through the
+additions to Erlang/OTP that I am most excited about!
+
+Erlang/OTP 24 includes contributions from 60+ external contributors totalling
+1390 commits changing 0.5 million(!) lines of code. Though I'm not sure the line
+number should count as we vendored all of [AsmJit] and re-generated the wxWidgets
+support.
```

Comment:
There isn't a standard for for, I'm using it here because lines added was pretty much the same as lines removed.

I've expanded the text a bit so provide some more context.